src/main/webapp/css/main.css

```diff
@@ -161,6 +161,83 @@ body {
   margin-top: 10px;
 }
 
+.messageContainer{
+  width:100%;
+  margin-bottom:10px;
+  overflow:hidden;
+  display:flex;
+}
+
+.right{
+  justify-content: flex-end;
+}
+
+.box {
+  background: #ffffff;
+  color: #007bff;
+  max-width: 30%;
+  margin-left:25px;
+  margin-top:10px;
+  margin-right:25px;
+  padding: 20px;
+  text-align: center;
```

Comment:
I prefer `text-align: left` but `center` looks good too.

Reply:
I agree

Reply:
You're right, Looks much better.